src/ProjectNeon/Assets/Features/Effects/EvadeAttacks.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using UnityEditor;
+
+public sealed class EvadeAttacks : Effect
+{
+    private Effect _effect;
+
+    public EvadeAttacks(int quantity) 
```

Comment:
Do we need this alias class?

If we do, then should be following proper `ctor` EO rules. Save the argument. Don't hide any factory code.

Reply:
@SilasReinagel Removed it